src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Controllers/UsersController.cs

```diff
@@ -28,7 +29,9 @@ public UsersController(IUserService userService)
         /// <returns> The <see cref="ActionResult"/>. </returns>
         public ActionResult Index()
         {
-            var users = _userService.GetAllUsers().ConvertAll(UserViewModel.Initialize);
+            //var users = _userService.GetAllUsers().ToList().ConvertAll(UserViewModel.Initialize);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)